ansible/roles/test/templates/fib.j2

```diff
@@ -2,20 +2,22 @@
 {% for podset in range(0, podset_number) %}
 {% for tor in range(0, tor_number) %}
 {% for subnet in range(0, tor_subnet_number) %}
-192.168.{{ podset }}.{{ tor * 16 + subnet }}/32 {% for ifname, v in minigraph_neighbors.iteritems() %}{% if "T2" in v.name %}{{ alias_reverse_map[ifname] }},{% endif %}{% endfor %}
+192.168.{{ podset }}.{{ tor * 16 + subnet }}/32 {% for ifname, v in minigraph_neighbors.iteritems() %}{% if "T2" in v.name %}{{ '%02d' % (alias_reverse_map[ifname]|replace("Ethernet","")|replace("fortyGigE0/", "")|int / 4)}},{% endif %}{% endfor %}
```

Comment:
filter for "fortyGigE/0" is not needed since you already applied the reverse map. the interface name are standardized to Ethernet{n}.